### DISCUSSION TITLE
Failed while sponsoring

### DISCUSSION THREAD

**Discussion by Akbaroke** [community/community#145607] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Why do I always fail when sponsoring? I have done it 2 times and it all failed. It shows a failed notification on the payment method. Is there a solution?

**Comment by IKHINtech** [comment, marked as answer]
what the payment method did you used ?

**Reply by Akbaroke** [reply]
okay thank you ikhin, all problems have been resolved
